protected/components/WebUser.php

```diff
@@ -27,13 +27,19 @@ public function checkAccess($operation, $params = array(), $allowCaching = true)
         return ($operation === $role);
     }
 
-    function getFirst_Name()
+    public function getFirstName()
     {
         $user = $this->loadUser(Yii::app()->user->id);
         return $user->first_name;
     }
 
-    function getEmail()
+    public function getLastName()
+    {
+        $user = $this->loadUser(Yii::app()->user->id);
```

Comment:
since you are extending CWebUser, you can access the current user directly via `$this` so there is no need to call `Yii::app()->user->id`, just use `$this->id`